tools/osbuilder/rootfs-builder/rootfs.sh

```diff
@@ -539,9 +539,10 @@ EOT
 	AGENT_DEST="${AGENT_DIR}/${AGENT_BIN}"
 
 	if [ -z "${AGENT_SOURCE_BIN}" ] ; then
+		[ "$ARCH" == "ppc64le" ] && [ "$LIBC" == "gnu" ] || die "LIBC type for ppc64le should be gnu"
 		bash ${script_dir}/../../../ci/install_musl.sh
-		# rust agent needs ${arch}-unknown-linux-musl
-		rustup show | grep linux-musl > /dev/null || bash ${script_dir}/../../../ci/install_rust.sh
+		# rust agent needs ${arch}-unknown-linux-${LIBC}
+		rustup show | grep linux-${LIBC} > /dev/null || bash ${script_dir}/../../../ci/install_rust.sh
```

Comment:
(For a later iteration)

This `rustup` check won't work on Fedora.

Reply:
And more for later iteration ... we shouldn't be installing musl if LIBC is gnu, neither adding it to the PATH.

Again, this is not related to this patch, but it's around the changed lines, which caught my attention.